planet-dumper.py

```diff
@@ -193,7 +193,7 @@ def upload_to_s3(file_path, bucket_name, dump_key):
             raise ObjectKeyAlreadyExists(f'Object key: {dump_key} already exists on the bucket: {bucket_name}')
 
         # upload dump
-        bucket.upload_file(file_path, dump_key)
+        bucket.upload_file(file_path, dump_key, ExtraArgs={'ACL':'public-read'})
```

Comment:
lets make the ACL configurable and have public-read as default

Reply:
done.